<launch>
  <arg name="host" default="192.168.0.1" />
  <master auto="start"/>
  <node pkg="lms1xx" name="lms1xx" type="LMS1xx_node" output="screen">
  <param name="host" value="$(arg host)" />
  <param name="frame_id" value="base_scanner" />
  </node>
  <node pkg="robot_setup_tf" name="tf_broadcaster" type="tf_broadcaster" output="screen">
  </node>
  <node pkg="joy" name="joy_node" type="joy_node" output="screen">
  </node>
  <node pkg="donkey_rover" name="rover_drive" type="rover_drive" output="screen">
  </node>

  <!-- point cloud creator-->
  <!--node pkg="laserscan_to_pointcloud" name="scanToPoint" type="scanToPoint" output="screen"-->
  <!-- /node -->
</launch>
